### DISCUSSION TITLE
Enclosure / CAD Assembly

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#20] [Show and tell]
Hey all, just wanted to show off the CAD assembly for my tester unit.    I have a lot of Makerbeam 10x10mm extrusion so decided I'd use that for my enclosure.  Ended up finding pre-made cad for the LCD2004 unit, led fan unit, and of course manufacture 3D cad drawings for the PSU and iec plug.

The goal was as compact of a unit I could comfortably make.   I gave up the ergonomics of having the two GX16x6 sensor plugs on the front in favor of smaller space.   The case measures 200x102x200.   I had to make 1mm spacers for the makerbeam joints to get the light unit to fit within the frame.   

Here's a couple screenshots of the assembly.     

![image](https://user-images.githubusercontent.com/12026545/227119329-96b5a67f-8c6f-42df-9f31-ea58dd1e340b.png)

![image](https://user-images.githubusercontent.com/12026545/227119298-a65181be-32c0-4e7f-8422-77dcd424426d.png)

I'm using a meanwell RS-25-12  (12v 2.1a) power unit, typical IEC snap-in power connector with on/off switch.  
The rest is from the BOM unit.

I'll share progress on the actual physical build once it's finished.

**Comment by ic-racer** [comment]
I'll be following along. 

One stumbling block I always encounter is making the rectangular opening for the IEC, so I usually make a round hole with a strain relief bushing. How will you make the opening? Will you 3d print the rear panel?

Thanks

**Reply by polygonfuture** [reply]
I'm 3D printing the panels.  Otherwise I'd probably use either laser cutting service or traditional route of paper template, drill bits for drilling out corner areas, and then a small coping saw to cut out the shape and then finally using a file to do finishing edges.  Last method is way more work but it does the job.

**Reply by ic-racer** [reply]
Following....

**Reply by srozum** [reply]
> traditional route of paper template, drill bits for drilling out corner areas, and then a small coping saw to cut out the shape and then finally using a file to do finishing edges. 

Yep, that's exactly how I did it. Those blue enclosures made of pretty soft metal and easy to work with.

**Comment by User-462** [comment]
That is a nice looking case.

**Comment by User-462** [comment]
In order to adjust the power supply voltage to calibrate the light source, I've been wondering about ways to avoid opening the case to do that. My current supply has the control built into the internal pcb. I've seen some power supplies that have the control on a separate potentiometer that could be detached and relocated. That would be one option. I've even considered a plastic rod glued to the built in pot that would protrude through the front panel of the case. 
And I suppose another way would be to add a separate voltage controller with a voltage display and mount it on the case somewhere. Would need to find a controller with the proper heatsinks.

Anyone solved this before?
